src/store/toast.store.ts

```diff
@@ -0,0 +1,97 @@
+type ToastMessage = {
+  type: 'success' | 'error'
+  message: string
+  id: number
+  show: boolean
+}
+
+type ToastState = {
+  messages: ToastMessage[]
+}
+
+const toastTimeout = 5000
```

Comment:
We only really need this constant. It's fine if the toast timeout includes the time for enter and exit.

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/98/commits/3f1cad69651fbd60506189a8eb2a21c8b69fa2a4